examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -117,6 +117,43 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
       }
       return filter;
     }
+    
+    function getSampleFilter(parameters, maxDay) {
```

Comment:
Please put a comment about this function: what does this function do? And I'm not sure why this function is required. Can you explain?

Reply:
@ChankiWu This function should not be inside your submission this time, which I think it's for heatmap's usage. please submit only the necessary code for your part works. Namely, the 3 icons to show and the "maptype" value changed by click on each icon.